tests/deploy/values/ci-sa-agent.yml

```diff
@@ -8,3 +8,9 @@ testData:
   marker: post_merge_agent
   agentName: elastic-agent
   clusterType: vanilla
+
+elasticsearch:
+  imageTag: "8.6.0-SNAPSHOT"
+
+kibana:
+  imageTag: "8.6.0-SNAPSHOT"
```

Comment:
> > @gurevichdmitry I'll create a ticket for it. I think we should try to find a more stable way to query for the cloudbeat process. Like using the agent's health endpoints. It's a bit out of scope for this pr, btw it's just the ci run - running this locally will take some more work.
> 
> We still can't merge it if it fails, as it will break our CI (once we fix the exit status issue) so we need to skip that part, and track it in the new issue

It won't break anything, just don't set this specific test as required until that test is set to skip.

Reply:
added a hermit env var, that takes versions from cloudbeat and passes it to just.
So it will always be current.